test/Microsoft.ML.AutoML.Tests/Microsoft.ML.AutoML.Tests.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <RootNamespace>Microsoft.ML.AutoML.Test</RootNamespace>
```

Comment:
You shouldn't need `RootNamespace` or `ProjectGuid` in this .csproj.

`RootNamespace` gets defaulted to the project name, and since it already matches, this is redundant.
`ProjectGuid` is unnecessary in SDK-style .csproj.